configure.ac

```diff
@@ -4812,66 +4761,65 @@ if test "$with_aint_size" -gt 0 ; then
         AC_MSG_RESULT([Overriding MPI_Aint to be $aint_size bytes])
    fi
```

Comment:
@pavanbalaji Should we remove the option `--with-aint-size`?

Reply:
I recommend that.  @roblatham00 what do you think?